pyproject.toml

```diff
@@ -212,6 +213,154 @@ module = [
 ]
 disallow_any_generics = false
 
+[[tool.mypy.overrides]]
+module = [
+    # tests/
+#    "tests.conftest",
+#    "tests.test_addnodes",
+    "tests.test_application",
```

Comment:
Note that by just commenting things like that, the number of lines do not change betweeen two branches and thus merge are orthogonal when they happen. In particular, it's much more easier to avoid merge conflicts.

Reply:
Now I'm not sure whether that's the case